### ISSUE TITLE
Merge types with type Subscription

### ISSUE THREAD

**Issue by tomyitav** [okgrow/merge-graphql-schemas#20]
Let's say I have the following types:

![train](https://cloud.githubusercontent.com/assets/16193919/25409541/06b93c94-2a1b-11e7-84a8-4de82da22448.PNG)

![cars](https://cloud.githubusercontent.com/assets/16193919/25409506/e1cd916e-2a1a-11e7-824b-eeaf0a5b0615.PNG)

and I want to concatenate them - 

![concat](https://cloud.githubusercontent.com/assets/16193919/25409577/2d5b74ca-2a1b-11e7-90bf-3bf1dc7f8ab2.PNG)
 
Then, when I use the schema in graphqlExpress in server.js file - I "lose" the Subscription types.
Is it possible to fix this?
Thanks!

**Comment by RodMachado** [comment]
Hey @tomyitav,

Please try using version 0.0.9. It should work for you. 

Let me know if it doesn't. 😁 

Cheers!

**Comment by tomyitav** [comment]
Thanks for the reply!
But I'm already using version 0.0.9. It isn't working...

**Comment by RodMachado** [comment]
Hey @tomyitav 

It's Fixed! Please use version 0.0.11.

Here's the output of your types with the latest version:

```schema {
    query: Query
    mutation: Mutation
    subscription: Subscription
  }

  type Query {
   car(name: String): [Car]  
   train(name: String): [Train]
  }

  type Mutation {
   updateCar ( currName: String! newName: String! ): Car
   addCar ( name: String! ): Car 
   deleteCar(name: String!): Car
  }

  type Subscription {
   carUpdated: Car 
   carAdded: Car 
   carDeleted: Car
  }
  
  type Car {
    _id: String
    name: String
  }

  type Train {
    _id: String
    name: String
    speed: Int
  }
```
Let me know if you run into any issues.

Cheers!

**Comment by tomyitav** [comment]
Great! Thanks @RodMachado !
